AdyenActions/Components/Await/AwaitComponent.swift

```diff
@@ -73,34 +75,52 @@ public final class AwaitComponent: ActionComponent, Cancellable {
     
     private let componentName = "await"
     
+    /// Handles redirect await action.
+    ///
+    /// - Parameter action: The await action object.
+    public func handleRedirectableAwait(_ action: AwaitAction) {
```

Comment:
If this function is also just taking an `AwaitAction` the logic to decide what to do could actually happen in the `AwaitComponent` instead of the `AdyenActionComponent` where you do `action.url == nil ? component.handle(action) : component.handleRedirectableAwait(action)` 

What do you think? 
This way we also don't have to add another function.